test/parser_test.exs

```diff
@@ -19,26 +19,61 @@ defmodule ParserTest do
   end
 
   test "parses attributes" do
-    {_, {:meta, opts}} = "meta name=variable content=\"one two\""
+    {_, {:meta, opts}} = ~S(meta name=variable content="one two")
                          |> Parser.parse_line
 
     assert opts[:attributes] == [content: "one two", name: {:eex, content: "variable", inline: true}]
   end
 
+  test "parses attributes with wrappers" do
+    {_, {:meta, opts}} = "meta(name=other content=\"one two\")"
+                         |> Parser.parse_line
+
+    assert opts[:attributes] == [content: "one two", name: {:eex, content: "other", inline: true}]
+
+    {_, {:meta, opts}} = ~S(meta {name=variable content="one two"})
+                         |> Parser.parse_line
+
+    assert opts[:attributes] == [content: "one two", name: {:eex, content: "variable", inline: true}]
+  end
+
+  test "parses boolean attributes" do
+    {_, {:input, opts}} = "input (type=\"text\" required=true)"
+                          |> Parser.parse_line
+
+    assert opts[:attributes] == [required: {:eex, content: "true", inline: true}, type: "text"]
+
+    {_, {:input, opts}} = "input (type=\"text\" required)"
+                          |> Parser.parse_line
+
+    assert opts[:attributes] == [required: {:eex, content: "true", inline: true}, type: "text"]
+  end
+
   test "parses attributes with interpolation" do
     {_, {:meta, opts}} = ~S(meta content="one#{two}") |> Parser.parse_line
 
     assert opts[:attributes] == [content: {:eex, content: ~S("one#{two}"), inline: true}]
   end
 
+  test "parses attributes with elixir code" do
```

Comment:
@Rakoth I updated the regex to accommodate attributes with elixir code.  I used your three examples as tests.